crates/ethereum-forks/src/hardfork/ethereum.rs

```diff
@@ -432,6 +432,10 @@ impl<const N: usize> From<[(EthereumHardfork, ForkCondition); N]> for ChainHardf
 }
 
 impl ConfigureHardforks for EthereumHardfork {
+    fn super_chain_mainnet_hardforks() -> impl Iterator<Item = (Self, ForkCondition)> {
+        Self::hardforks(Chain::mainnet()).iter().copied()
+    }
+
     fn init_block_hardforks(
```

Comment:
could we merge those into a single fn which would return correctly ordered iterator over `(Self, ForkCondition)`?

Reply:
not sure I follow?

Reply:
rn we have `init_block_hardforks`, `init_time_hardforks` and `init_paris` methods which we invoke separately and then order their combined output based on `super_chain_mainnet_hardforks`, I'm wondering what's the motivation behind splitting those?

after a second look I don't think we use `ordered_hardforks` and thus order of hardforks at all here?